easybuild/easyconfigs/s/SAGE/SAGE-6.3.eb

```diff
@@ -4,18 +4,18 @@
 
 easyblock = "Tarball"
 
-name = 'S.A.G.E.'
+name = 'SAGE'
 version = '6.3'
 
-homepage = 'https://code.google.com/archive/p/opensage/'
+homepage = 'http://darwin.cwru.edu/sage/'
 description = """ S.A.G.E. (Statistical Analysis for Genetic 
 Epidemiology) is free software package containing programs for 
 use in the genetic analysis of family, pedigree and individual data."""
 
 toolchain = {'name': 'dummy', 'version': 'dummy'}
 
-source_urls = ['https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/opensage/']
-sources = ['%(name)sv%(version)s_Linux_x86_64.tar.gz']
+source_urls = ['https://github.com/elstonsage/sageCore/releases/download/v6.3']
+sources = ['S.A.G.E.v%(version)s_Linux_x86_64.tar.gz']
```

Comment:
@rjeschmi let's include a checksum too, since we're changing the `source_urls`?

I have `c94a0b9af656212251ce452f7b2f99ec` for this one, both for old & new `source_urls`.